### PR TITLE
maintainers: add CompEng0001

### PR DESCRIPTION
**Pull request by CompEng0001**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/editors/vscode/extensions/default.nix [hunk 1]
```diff
@@ -48,6 +48,22 @@ let
...
+        };
+        meta = {
+          description = "Provide rainbow colors for the round brackets, the square brackets and the squiggly brackets. This is particularly useful for Lisp or Clojure programmers, and of course, JavaScript, and other programmer";
+          downloadPage = "https://marketplace.visualstudio.com/items?itemName=2gua.rainbow-brackets";
+          homepage = "https://github.com/lcultx/rainbow-brackets";
+          license = licenses.mit;
+          maintainers = with maintainers; [ CompEng0001 ];
+        };
+      };
+
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/vscode/extensions/default.nix:66]:
```suggestion

```

File: pkgs/applications/editors/vscode/extensions/default.nix [hunk 2]
```diff
@@ -48,6 +48,22 @@ let
         };
       };
 
+      _2gua.rainbow-brackets = buildVscodeMarketplaceExtension {
+        mktplcRef = {
+          publisher = "2gua"
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/vscode/extensions/default.nix:53]:
```suggestion
          publisher = "2gua";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please test your changes before submitting.

File: pkgs/applications/editors/vscode/extensions/default.nix (hunk 1 above)
Comment (CompEng0001) [reply pkgs/applications/editors/vscode/extensions/default.nix]: Removed.

File: pkgs/applications/editors/vscode/extensions/default.nix (hunk 2 above)
Comment (CompEng0001) [reply pkgs/applications/editors/vscode/extensions/default.nix]: Removed

**Comment by SuperSandro2000** [comment]
Please cherry-pick the commit into the PR (#189331) where you are adding yourself as a maintainer to a package. This makes sure all maintainers entries are actually used and we do not collect stale ones.

**Pull request closed without merging**